Made this a Jekyll project

jeffmaher opened the pull request:
Moved existing documents to the `docs` folder (Content and Style Guides).

This is just an empty Jekyll project, so it doesn't really do anything right now.

Commit:
Made this a Jekyll project

jeffmaher commented:
For issue #20

Commit:
Merge branch 'master' into jekyllize

awong-dev commented on _config.yml:
```diff
@@ -0,0 +1,21 @@
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Your awesome title
```
Can we add `source: _source` or something so the top-level directory containing things like .gitignores, etc., don't get mixed into the jekyll tree?

I was about to make that change for vets-website too.

plusjeff replied:
Yep! Added.

awong-dev replied:
oh...you need to add a jekyll_source directory too.

awong-dev replied:
and then move all the jekyll template goo into it.

awong-dev replied:
pushed 1a97286 to fix.

Commit:
Merge branch 'master' into jekyllize

Commit:
Added Jekyll source

emilyville commented:
I can't seem to run this locally - also I suggest we add a readme - this one has some basic stuff that we could port over since I think they are both stock!

https://github.com/department-of-veterans-affairs/vets.gov-status

awong-dev committed:
Move all jeykyll files into src.

Also decided to name the template directory `src` cause it's shorter and
since this is a jekyll site, jekyll is implied.

awong-dev commented:
@emilyville that was due to the config change I suggested fro Jeff.  My lastest push fixes it all.

emilyville commented:
LGTM!
